04 Strategy Library/207 Value Effect within Countries/02 Method.html

```diff
@@ -0,0 +1,87 @@
+<p>
+  This new metric is called the cyclically adjusted price-to-earnings ratio (CAPE). The basic idea is comparing the stock prices with earnings smoothed across multiple years.
```

Comment:
CAPE isn't new:
"The CAPE (cyclically adjusted price-to-earnings ratio) compares stock prices with earnings smoothed across multiple years."